src/components/editor/ArticleListController.jsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { browserHistory } from 'react-router';
+import FalcorController from "lib/falcor/FalcorController";
+import List from "./List";
+
+var pageLength = 50;
```

Comment:
why is this outside, is this supposed to be a constant. Also it's name is not clear, maybe something like

`DEFAULT_NUM_ARTICLES_SHOWN`